src/modules/chuck-norris/categories.ts

```diff
@@ -0,0 +1,42 @@
+import { InternetModule } from "@faker-js/faker"
+
+const facts = require("./api/chuckfacts.json")
+
+// const array = [{
+//     "name": "jane",
+//     "age": 17
+//   },
+//   {
+//     "name": "joe",
+//     "age": 17
+//   },
+//   {
+//     "name": "bob",
+//     "age": 35
+//   }
+// ];
+// const uniques = array.map(item => item.age)
+//   .filter((value, index, self) => self.indexOf(value) === index)
+// console.log(uniques)
+
+ 
+let uniqueCatgories = new Map<string, string>();
+facts.map(
+  fact => { 
+    fact.categories.map(
+      category =>{
+        if(!uniqueCatgories.has(category)){
+          uniqueCatgories.set(category,"")
+        }
+      })  
+    } )
+
+console.log(Array.from(uniqueCatgories.keys()))
```

Comment:
Same rule on console.logs as mentioned before.

Reply:
this categories can go - it's not required (and was for testing my map method)

Reply:
have removed the file - it was not required.